### PR TITLE
tetragon: Add uprobe multi support

### PR DESCRIPTION
**Pull request by olsajiri**
cilium/ebpf changes https://github.com/cilium/ebpf/pull/1269

### CODE REVIEW

File: pkg/bpf/detect.go [hunk 1]
```diff
@@ -70,6 +72,47 @@ func HasKprobeMulti() bool {
...
+		return false
+	}
+	defer prog.Close()
+
+	ex, err := link.OpenExecutable("/proc/self/exe")
+	if err != nil {
+		return false
+	}
+
+	// need cilium/ebp fix, can't pass just addresses without symbol
```
Comment (kkourt) [inline pkg/bpf/detect.go:100]: Is there an issue for this on cilium/ebpf? Would be nice to have a reference to check when this is fixed and update the code.

nit: typo -> cilium/ebp

**Review by kkourt — APPROVED** [review]
Looks good to me, thanks!

**Review by jrfastab — APPROVED** [review]

**Comment by jrfastab** [comment]
Looks good fix small typo and merge imo.

File: pkg/bpf/detect.go (hunk 1 above)
Comment (olsajiri) [reply pkg/bpf/detect.go:100]: ah, the comment is no longer valid, it got fixed as preparation for uprobe multi support in ebpf/cilium lib
now it is:

```
func (ex *Executable) addresses(symbols []string, addresses, offsets []uint64) ([]uint64, error) {
        n := len(symbols)
        if n == 0 {
                n = len(addresses)
        }
```
will remove the comment and fix the typo, thanks

**Pull request merged**